lib/active_reporting/database_adapters/factory.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module ActiveReporting
+  module DatabaseAdapters
+    class Factory
+      class << self
+        ADAPTERS = {
+          sqlite3: SqliteAdapter,
+          mysql2: MysqlAdapter,
```

Comment:
Let's go with just `sqlite` and `mysql` here. The use of "3" and "2" in various Rails projects references the adaptor and version of the gem they use. In the case of this gem, we shouldn't care.